core/src/main/java/com/datastrato/gravitino/storage/relational/mapper/SchemaMetaMapper.java

```diff
@@ -120,7 +120,7 @@ SchemaPO selectSchemaMetaByCatalogIdAndName(
           + " AND schema_name = #{oldSchemaMeta.schemaName}"
           + " AND metalake_id = #{oldSchemaMeta.metalakeId}"
           + " AND catalog_id = #{oldSchemaMeta.catalogId}"
-          + " AND schema_comment = #{oldSchemaMeta.schemaComment}"
+          + " AND (schema_comment IS NULL OR schema_comment = #{oldSchemaMeta.schemaComment})"
```

Comment:
This is to fix the problem that `schema_comment` is nullable

Reply:
I'm wondering if it is necessary for us to convert the `null` value to `""` while inserting, to solve this problem

Reply:
Should we set this parameter to `Not Null` and use `""` as the default value?

Reply:
Comment null and empty are different for many catalogs, so I don't think it's better than the previous design.